packages/headless/src/features/generated-answer/generated-answer-state.ts

```diff
@@ -1,12 +1,37 @@
 import {GeneratedAnswerCitation} from '../../api/generated-answer/generated-answer-event-payload';
 
 export interface GeneratedAnswerState {
+  /**
+   * Determines if the generated answer is loading.
+   */
   isLoading: boolean;
+  /**
+   * Determines if the generated answer is streaming.
+   */
   isStreaming: boolean;
+  /**
+   * The generated answer.
+   */
   answer?: string;
+  /**
+   * The citations used to generate the answer.
+   */
   citations: GeneratedAnswerCitation[];
+  /**
+   * Determines if the generated answer is liked, or upvoted by the end user.
+   */
   liked: boolean;
+  /**
+   * Determines if the generated answer is disliked, or downvoted by the end user.
+   */
   disliked: boolean;
+  /**
+   * Determines if the feedback modal with the purpose of explaining why the end user disliked the generated answer is currently opened.
```

Comment:
```suggestion
   * Determines if the feedback modal is currently opened with the purpose of explaining why the generated answer was disliked by the user.
```

I think this description is a bit more clear what do you think?